src/controllers/pods.ts

```diff
@@ -223,4 +248,10 @@ export default function (server: Hapi.Server, deps: Injector) {
       }
     }
   })
+
+  server.route({
+    method: 'GET',
+    path: '/pods/{id}/logs',
+    handler: getPodLogs
```

Comment:
The param should probably be validated to exist. I believe this can be done via joi see https://hapijs.com/tutorials/validation
```javascript
options: {
        validate: {
            params: {
                id: Joi.string()
            }
        }
    }
```
Unless hapi is already doing validation on this somehow.

Reply:
I think since it's part of the URL it's guaranteed to exist and be a string, otherwise the URL would not have matched.

Reply:
Good point, plus even if its not right or its a number it will fail on the getPods and reply back with the error.